silence-removal/README.md

```diff
@@ -0,0 +1,63 @@
+# Adaptive voice activity detection
+Python script that cuts out silent audio sounds. [Original `vad.py`](https://github.com/mauriciovander/silence-removal) written by [mauriciovander](https://github.com/mauriciovander/)
+
+## Table of contents
+- [Adaptive voice activity detection](#adaptive-voice-activity-detection)
+  - [Table of contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+    - [Directories](#directories)
+  - [Usage](#usage)
+  - [Details](#details)
+  - [References:](#references)
+
+--
+
+## Prerequisites
+The system must have [ffmpeg](https://ffmpeg.org/download.html) installed. On MacOS, this can be installed with [brew](https://formulae.brew.sh/formula/ffmpeg) via `brew install ffmpeg`.
+
+### Directories
+`vad.py` will generate a logging folder in its directory and generates the output to its directory as well. `useful_functions.py` and `convert_to_wav.py` should be in the same directory as `vad.py`.
+
+## Usage
+Examples of using the silence removal script `vad.py`:
+```{python}
+python vad.py './data/0no4f8nb0d351.aac' '0no4f8nb0d351_(processed)'
+python vad.py './data/clip_(stereo).mp3' 'clip_(stereo)_(processed)'
+python vad.py './data/taps_(mono).wav' 'taps_(mono)_(processed)'
```

Comment:
you have a log instead of a print, but maybe mention what will be in logged on a successful run, so people know what to look for?

Reply:
I've added log file info into the [Readme](https://github.com/myHaiven/data-collection/blob/c6b649171ef3d22a3ee9eaf40b0c12d1f0e2f13c/silence-removal/README.md#details):
![Log info](https://user-images.githubusercontent.com/46179386/122506131-85bfcd00-cfb2-11eb-9c12-b2181db7749d.png)